.github/workflows/docker.yml

```diff
@@ -0,0 +1,84 @@
+on:
+  schedule:
+    - cron: '0 7 */7 * *' # This should run once a week
+
```

Comment:
I would re-run the build if the image file/this file gets modified on master branch only

Reply:
I think is not really possible :thinking:

Last time I tried on Replay, GitHub screamed that the file was invalid and I had to remove it https://github.com/nahuelwexd/Replay/commit/fddf31b221d1f70aef9a71ac32dee66ea26fc457

Reply:
Then maybe schedule it each 1 hour so we can get the time to ensure things are working & we can switch back to the weekly job

Reply:
Okay